core/src/main/java/org/geysermc/geyser/configuration/GeyserJacksonConfiguration.java

```diff
@@ -91,9 +91,6 @@ public abstract class GeyserJacksonConfiguration implements GeyserConfiguration
     @JsonProperty("debug-mode")
     private boolean debugMode = false;
 
-    @JsonProperty("allow-third-party-capes")
-    private boolean allowThirdPartyCapes = true;
```

Comment:
It *might* be interesting to keep these for a few versions (without any functionality), and warn in the startup log that we've migrated these options to an extension - just a thought though

Reply:
If we have an extension ready, maybe we can just say "please download this extension" if the config option is enabled and keep the config option as a dummy?